.github/workflows/build-operator-images.yml

```diff
@@ -3,7 +3,7 @@ name: Build and Publish RSCT Operator
 on:
   push:
     tags:
-      - '[0-9]+.[0-9]+.[0-9]+'
+      - 'v*.*.*'
```

Comment:
The Operator SDK does not support the 'v' character in version numbers; therefore, used versions without 'v'. To address this, we can modify -
 https://github.com/ocp-power-automation/rsct-operator/blob/a00303edac604becddcef157256e35ed38178a7a/.github/workflows/build-operator-images.yml#L19 this to

```
echo "VERSION=$(echo ${GITHUB_REF/refs\/tags\/v/})" >> $GITHUB_ENV
```

Reply:
okay, lemme do that then..

Reply:
done that change, ptal to the latest codechange

Reply:
/lgtm